src/evaluators/state/PHAL_LoadSideSetStateField_Def.hpp

```diff
@@ -99,7 +99,7 @@ evaluateFields(typename Traits::EvalData workset)
   int size = dims.size();
   // Check the tag of the first extent after (cell,side,) to determine if field is nodal
   const std::string& leading_field_tag = size>2 ? field.fieldTag().dataLayout().name(2) : "";
-  TEUCHOS_TEST_FOR_EXCEPTION (size>2 && leading_field_tag!="Node" && leading_field_tag!="Dim" && leading_field_tag!="VecDim", std::logic_error,
+  TEUCHOS_TEST_FOR_EXCEPTION (size>2 && leading_field_tag!=PHX::print<Node>() && leading_field_tag!=PHX::print<Dim>() && leading_field_tag!="PHX::print<VecDim>()", std::logic_error,
```

Comment:
I think you left some spurious quote marks for VecDim.

Reply:
Good catch!